docker-compose-devel.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
I have a couple notes about this docker dev file (and local development in general):

1. `authserver` already has a docker-compose-development.yml file in the `Docker` directory. The existing one also spins up the Flask app. Can you delete this newly added one? 
2. Using Docker `build` plus `docker-compose` I am unable to stand up the app with the new migrations. 

```
docker build -t brighthive/authserver:latest .
docker-compose -f Docker/docker-compose-development.yml up
```
Why not? A couple reasons: (1) the `cmd.sh` file does not run migrations for `DEVELOPMENT` or an empty environment, and (2) migration `1794e645d265` does not account for a `DEVELOPMENT` environment. Can we adjust the `cmd.sh` and migration file to allow for `DEVELOPMENT`? 

Alternatively, can you document hove you've been spinning up Authserver for local development?

Reply:
`flask db ugrade`? Alternatively you can use the `manager.py` to do that. `python manager.py db upgrade`. I didn't put documentation here for two reasons 1) I sort of assumed it was understood how to use Flask-Migrate to upgrade a migration. 2) I have a bigger task to actually document authserver on tap for this sprint.

Reply:
Yes, but I want to run my Flask app in a container, and I do not want to shell into that container and run commands. I'd like to just run docker-compose and be on my way.